### DISCUSSION TITLE
Backup & Restore Honeygain

### DISCUSSION THREAD

**Discussion by superit63** [engageub/InternetIncome#133] [Q&A]
Hello,
How can I backup Honeygain app in my old PC and move it to new PC? I got issue when I run in new PC with the same proxies because of device limitation.
Thanks

**Comment by engageub** [comment]
Hello,

Honeygain relies on device name.
Could you please check if you have given same device names.

Thank you

**Reply by superit63** [reply]
Hi,

You mean DEVICE_NAME='ubuntu' as the default value. 
I have changed device name to easier manage. So, I just need keep the Device name the same and proxy the same in my new PC?

Thanks

**Comment by engageub** [comment]
Yes, you need to use the same device name as mentioned earlier.
